### ISSUE TITLE
ConnectionError on Ubuntu 22.04

### ISSUE THREAD

**Issue by Gwenn-LR** [ddetommaso/tobiiglasses-controller#41]
Hi,

First of all, thanks for this work which seems promising.

However I struggle to connect my Tobii Pro Glasses 2 on my Ubuntu.

Official User Manuel does not indicate a compatibility with Linux but some [official articles](https://connect.tobii.com/s/article/What-software-and-operating-systems-are-compatible-with-Tobii-Pro-eye-trackers?language=en_US) indicates it should.

I was wondering if you could share me some indications so I can use it and connect trrough your repo.

Thanks for the attention you'll bring to my question.

**Comment by Gwenn-LR** [comment]
I have finally managed to get it running by connecting it to a Windows to access the [Tobii Pro Glasses Controller](https://connect.tobii.com/s/g2-downloads?language=en_US) in order to activate the Wifi.

Then I have connected to it trough the according SSID and the default password (`TobiiGlasses`).

Finally I have been able to get the device with:
```python
tobiiglassesctrl.TobiiGlassesController("192.168.71.50")
```

And your package works like a charm!

**Comment by Gwenn-LR** [comment]
I have finally managed to get it running by connecting it to a Windows to access the [Tobii Pro Glasses Controller](https://connect.tobii.com/s/g2-downloads?language=en_US) in order to activate the Wifi.

Then I have connected to it trough the according SSID and the default password (`TobiiGlasses`).

Finally I have been able to get the device with:
```python
tobiiglassesctrl.TobiiGlassesController("192.168.71.50")
```

And your package works like a charm!
